ddc/developer-guide/quickstart.md

```diff
@@ -41,6 +36,12 @@ _**package.json**_
   "type": "module"
 }
 ```
+
+Then run:
+```
+npm install --save-dev typescript
```

Comment:
Do we need typescript? We have on JS example

Reply:
Don't think so - just didn't remove it from the previous version. Now will remove

Reply:
Done